sdk/src/Core/Amazon.Runtime/Internal/Util/JsonConstants.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amazon.Runtime.Internal.Util
+{
+    /// <summary>
+    /// Constants used in JSON unmarshalling.
+    /// </summary>
+    public static class JsonConstants
+    {
+        public static ReadOnlySpan<byte> Utf8Bom => new byte[] { 0xEF, 0xBB, 0xBF };
```

Comment:
I can't do this because `ReadOnlySpan<byte>` is a ref struct and can't be used as an auto-implemented property unless the class itself is a ref struct :(

Reply:
got it 👍